test/conformance/helpers/tracing/traces.go

```diff
@@ -62,29 +64,70 @@ func (t SpanTree) toTestSpanTreeHelper() TestSpanTree {
 	for i := range t.Children {
 		children[i] = t.Children[i].toTestSpanTreeHelper()
 	}
-	return TestSpanTree{
+	tst := TestSpanTree{
 		Kind:                     t.Span.Kind,
 		LocalEndpointServiceName: name,
 		Tags:                     t.Span.Tags,
 		Children:                 children,
 	}
+	tst.SortChildren()
+	return tst
 }
 
 // TestSpanTree is the expected version of SpanTree used for assertions in testing.
 type TestSpanTree struct {
-	Root                     bool
-	Kind                     model.Kind
-	LocalEndpointServiceName string
-	Tags                     map[string]string
+	Note                     string            `json:"aaNote,omitempty"`
+	Root                     bool              `json:"root,omitempty"`
+	Kind                     model.Kind        `json:"aKind,omitempty"`
+	LocalEndpointServiceName string            `json:"aName,omitempty"`
+	Tags                     map[string]string `json:"tags,omitempty"`
 
-	Children []TestSpanTree
+	Children []TestSpanTree `json:"zChildren,omitempty"`
 }
 
 func (t TestSpanTree) String() string {
 	b, _ := json.Marshal(t)
 	return string(b)
 }
 
+func (t *TestSpanTree) SortChildren() {
```

Comment:
Can we have some comments for what this sort function does?

Reply:
Done.